csharp/test/Ice/ami/AllTests.cs

```diff
@@ -620,14 +620,18 @@ public static void Run(TestHelper helper, bool collocated)
             }
             if (p.GetConnection() != null)
             {
-                // Stress test cancellation to ensure we exercise the various cancellation points.
+                // Stress test cancellation to ensure we exercise the various cancellation points. Cancellation of
+                // the sleep might fail or succeed on the server side depending how long we sleep.
+                var cancelCtx = new Dictionary<string, string> { { "cancel", "mightSucceed" } };
                 for (int i = 0; i < 20; ++i)
                 {
                     var source = new CancellationTokenSource();
                     source.CancelAfter(TimeSpan.FromMilliseconds(i));
                     try
                     {
-                        p.Clone(connectionId: $"cancel{i}").SleepAsync(50, cancel: source.Token).Wait();
+                        p.Clone(connectionId: $"cancel{i}").SleepAsync(50,
```

Comment:
Why are we using connectionId for this cancellation test? I assume canceling a request does not terminate the connection.

Reply:
Good question... it might have been useful at some point but it's not anymore, I removed it.